<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trainingview</class>
 <widget class="QWidget" name="trainingview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <item>
      <widget class="QLabel" name="labelPlans">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Wybierz plan do realizacji:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWorkouts">
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <layout class="QHBoxLayout" name="headerRight">
       <item>
        <widget class="QLabel" name="labelDetails">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Wprowadź swoje wyniki:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblDate">
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="displayFormat">
          <string>yyyy-MM-dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableDetails">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <column>
        <property name="text">
         <string>Ćwiczenie</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Typ</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Serie / Czas (min)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Powt. / Dystans (km)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ciężar (kg)</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveWorkout">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2E7D32; color: white; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>ZATWIERDŹ I ZAPISZ TRENING</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
